orbslam3:
  ros__parameters:
    color_topic: "/camera/color/image_raw"
    depth_topic: "/camera/depth/image_raw"

    world_frame: "map"
    odom_frame: "odom"
    camera_frame: "camera_color_frame"
    local_mapping: false

    queue_size: 5
    pose_use: "VO"